cli/client.go

```diff
@@ -380,6 +387,83 @@ func RobotsPartShellAction(c *cli.Context) error {
 	)
 }
 
+// CheckUpdateResponse holds the values used to hold release information.
+type CheckUpdateResponse struct {
+	Name       string `json:"name"`
+	TagName    string `json:"tag_name"`
+	TarballURL string `json:"tarball_url"`
+}
+
+// CheckUpdate is the corresponding Action for 'check-update'.
+func CheckUpdate(c *cli.Context) error {
```

Comment:
```suggestion
func CheckUpdateAction(c *cli.Context) error {
```
[nit] for consistent naming with other action functions.